lms/views/api/grading.py

```diff
@@ -88,7 +102,9 @@ def record_canvas_speedgrader_submission(self):
         self.lti_grading_service.record_result(
             lis_result_sourcedid, pre_record_hook=CanvasPreRecordHook(self.request)
         )
-
+        self.request.registry.notify(
+            LTIEvent(request=self.request, type=LTIEvent.Type.SUBMISSION)
```

Comment:
For the submissions there's not really any interesting data about it other than it did happen and when it did.